client/js/gdpr/chart/index.js

```diff
@@ -246,6 +252,25 @@ function createDefs() {
   return defs;
 }
 
+function createGrid(dates) {
+  const grid = document.createElementNS(SVG_NS, 'g');
+  grid.classList.add('grid');
+
+  for (let i = 0; i < dates.length; i++) {
+    const verticalLine = document.createElementNS(SVG_NS, 'line');
+    verticalLine.classList.add('date-line');
+
+    verticalLine.setAttribute('x1', RUN_SCALE_X * i + 'px');
+    verticalLine.setAttribute('x2', RUN_SCALE_X * i + 'px');
+    verticalLine.setAttribute('y1', '0px');
+    verticalLine.setAttribute('y2', RUN_SCALE_Y * 15 + 'px');
```

Comment:
good use of magic nums (what is `15` doing here though?) 

Add constant for 15

Reply:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/line helps me understand what's going on